webpack.config.js

```diff
@@ -1,7 +1,12 @@
 const path = require('path');
+const buildDirectory = './public/dist';
+
 module.exports = {
-    entry: {
-        bundle: path.join(__dirname, './client/sizeGrid.jsx')
+    entry: './client/sizeGrid.jsx',
+    externals: {
+      'cheerio': 'window',
+      'react/lib/ExecutionEnvironment': true,
+      'react/lib/ReactContext': true,
```

Comment:
Don't understand why you're excluding these modules from your bundle file.  Can't find them anywhere in package.json file.